modules/codegen/src/smithy4s/codegen/internals/SmithyToIR.scala

```diff
@@ -1025,6 +1025,53 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
       }
     }
 
+    def getFieldsPlain = {
```

Comment:
**thought:** this is just copied `fields` method with a single difference that it does not call `hints(member)` inside. Once we agree that this is the correct solution we can deduplicate code between these two methods. I haven't done it deliberately to make it clearer where the problem was and also not to spend to much time on PoC solution.